content/700-optimize/400-recommendations/100-excessive-number-of-rows-returned.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Excessive number of rows returned'
+metaTitle: 'Optimize Recommendations: Excessive number of rows returned'
+metaDescription: "Learn about the recommendation provided by Optimize for excessive number of rows returned."
+tocDepth: 3
+toc: true
+---
+
+The following query targeting a `User` model do not provide a [`take` option](/orm/reference/prisma-client-reference#findmany).
+
+```ts
+await user.findMany({ select: { email: "niko@prisma.io", }})
```

Comment:
prolly shouldnt use nikos email no?

Reply:
that's not niko's email, it's a fake one 😄 .

Reply:
I would recommend to include **prisma** in the query as our convention:
```ts
await prisma.user.findMany({ select: { email: "niko@prisma.io"  }})
```
Also removed the trailing comma.